### ISSUE TITLE
"Viewer" can not pull images

### ISSUE THREAD

**Issue by vanga** [SUSE/Portus#300]
Portus git commit hash `07063df983a09cd72f912b0ac27e858d729c4d48`
This worked initially. Didn't make any changes to the setup other than pushing/overriding the images (push with same tag).

Now its not working, changing the Role of the user to `Contributor` works though. 

corresponding portus_registry container logs
```
time="2015-09-01T06:40:18.905617084Z" level=error msg="error authorizing context: authorization token required" http.request.host=registry.example.com http.request.id=eb2d9bc9-d7ec-4b53-8046-8f98d55cbc1a http.request.method=GET http.request.remoteaddr="xxx" http.request.uri="/v2/" http.request.useragent="docker/1.8.1 go/go1.4.2 git-commit/d12ea79 kernel/3.13.0-61-generic os/linux arch/amd64" instance.id=5f2981fa-e8c7-423f-86b4-673733f08a4e version=v2.0.1 
time="2015-09-01T06:40:18.905693945Z" level=info msg="response completed" http.request.host=registry.example.com http.request.id=eb2d9bc9-d7ec-4b53-8046-8f98d55cbc1a http.request.method=GET http.request.remoteaddr="xxx" http.request.uri="/v2/" http.request.useragent="docker/1.8.1 go/go1.4.2 git-commit/d12ea79 kernel/3.13.0-61-generic os/linux arch/amd64" http.response.contenttype="application/json; charset=utf-8" http.response.duration=1.617554ms http.response.status=401 http.response.written=114 instance.id=5f2981fa-e8c7-423f-86b4-673733f08a4e version=v2.0.1 
xxx - - [01/Sep/2015:06:40:18 +0000] "GET /v2/ HTTP/1.1" 401 114 "" "docker/1.8.1 go/go1.4.2 git-commit/d12ea79 kernel/3.13.0-61-generic os/linux arch/amd64"
```

**Comment by mssola** [comment]
Did this work with a version of Docker previous to 1.8 ? I'm saying this because there's a known issue about this: #276. Basically, docker changed the authentication process a bit. Note that a fix is already on the works in this PR #282

**Comment by vanga** [comment]
@mssola You are right. My portus setup hasn't changed, but the docker client where I am doing docker pull is updated. 
I just checked this with docker 1.7, docker pull works as expected.

Thanks.

Closing this.
